src/cljs/athens/devcards/left_sidebar.cljs

```diff
@@ -43,13 +44,18 @@
   (with-styles +flex-column-align-start
     {:flex               "0 0 288px"
      :padding            "32px 32px 16px 32px"
-     :box-shadow         "1px 0 #433f3880"}))
+     :box-shadow         (str "1px 0 " (:panel-color COLORS))}))
 
 
 (def +left-sidebar-collapsed
   (with-styles +left-sidebar
-    {:flex "0 0 32px"
-     :padding "32px 0px"
+    {:flex "0 0 40px"
+     :align-items "flex-start"
+     :justify-content "flex-start"
+     :grid-template-rows "min-content"
+     :padding "32px 4px"
+     :display "grid"
+     :grid-gap "4px"
```

Comment:
I know it looks good in the app, but do we want to define height for collapsed sidebar? We might modify it down the line, in which full height would be nice

Reply:
Fixed in #144